### PR TITLE
baselayout/group: add group "render" for udev

### PR DESCRIPTION
**Pull request by t-lo**
This is a requirement for https://github.com/flatcar-linux/coreos-overlay/pull/543; please see that PR for description & instructions.

Addresses https://github.com/flatcar-linux/Flatcar/issues/169 in the initramfs.

### CODE REVIEW

File: baselayout/group [hunk 1]
```diff
@@ -45,5 +45,6 @@ portage:x:250:core
 tss:x:252:tss
 utmp:x:406:
 core:x:500:
+render:x:999:
```
Comment (krnowak) [inline baselayout/group]: Question: Why does this GID (999) not match GID passed in the portage-stable PR (which is 28 - https://github.com/flatcar-linux/portage-stable/pull/98/files#diff-b7353f2ecb7da8c1e846ba084605f940R8)?

Comment (t-lo) [reply baselayout/group]: I'm not entirely sure. 999 is the GID the group ends up in the OS image - I noticed when building the image locally. That's where I took the GID for `baselayout` from.

Comment (t-lo) [reply baselayout/group]: @krnowak this might explain it: https://github.com/flatcar-linux/coreos-overlay/pull/543#discussion_r476499498

Comment (sayanchowdhury) [reply baselayout/group]: https://github.com/gentoo/gentoo/blob/master/eclass/user.eclass#L277-L284 -- It loops through and picks the first free gid, that's what I understood.

Comment (t-lo) [reply baselayout/group]: And it starts at 999, looping backwards, which explains GID 999 being used in the end. I've updated https://github.com/flatcar-linux/coreos-overlay/pull/543 as well as  this PR to use GID 30.

**Review by krnowak — APPROVED** [review]
Looks jut.

**Pull request merged**